src/components/MoreInfoModal/PropertyInformationList.tsx

```diff
@@ -0,0 +1,55 @@
+import { List, ListItem, ListItemText } from '@mui/material';
+
+import styles from './MoreInfoModal.module.scss';
+
+interface PropertyInformation {
+  propertyType: string;
+  area: string;
+  yearBuilt: string;
+  floor: string;
+  floorsInBuilding: string;
+  roomsNumber: string;
+  bathroomNumber: string;
+  heating: string;
+}
+
+function PropertyInformationList(props: PropertyInformation) {
+  const { propertyType, area, yearBuilt, floor, floorsInBuilding, roomsNumber, bathroomNumber, heating } = props;
```

Comment:
How about replacing the step with spreading the props directly inside the brackets of `function PropertyInformationList(props: PropertyInformation)`?